authenticate/claims_test.go

```diff
@@ -25,13 +25,27 @@ var _ = Describe("Claims", func() {
 		}
 	)
 
+	DescribeTable("CheckDiaInstrument",
+		func(instrumentName1, instrumentName2 string, expected bool) {
+			// Use the checkDiaInstrument function from the imported package
+			Expect(claim.CheckDiaInstrument(instrumentName1, instrumentName2)).To(Equal(expected))
+		},
+		Entry("both diaA and diaB", "dia1234a", "dia5678b", true),
```

Comment:
Are we sure that this scenario is correct? Will users actually be going from a questionnaire 123 to 456? In the ticket it just says 

The questionnaires will always start with dia.
The primary questionnaire will be suffixed with a a
The secondary questionnaire will be suffixed with a b 

Should this actually return false?

Reply:
This was a good spot from James, and is now changed